### PR TITLE
Front page search and autocompletion

### LINKED ISSUE #233: Front page - Search bar + autocompletion

### PR DESCRIPTION
**Pull request by pgiraud**
Fixes #233.

### CODE REVIEW

**Comment by pgiraud** [comment]
Please review.

File: thinkhazard/static/less/common.less [hunk 1]
```diff
@@ -194,12 +192,11 @@ ul.horizontal li {
 .tt-suggestion:hover {
   cursor: pointer;
   color: #fff;
```
Comment (marionb) [inline thinkhazard/static/less/common.less]: If the background colour is white when hovering the text is almost not visible. Looking at the mock-up the text colour should stay the same as when not hovering.

Comment (pgiraud) [inline thinkhazard/static/less/common.less]: Good catch.
I only tested selection with the keyboard. My bad.

**Comment by marionb** [comment]
looks good, pleas merge.

**Pull request merged**